src/main/openapi/ds-storage-openapi_v1.yaml

```diff
@@ -66,7 +66,7 @@ paths:
           required: true
           schema:
             type: string
-            example: ${example_origin}
+            example: ${config:allowed_origins[0].name}
```

Comment:
Could this (an all the other similar usages) be an enum instead of a single value?